### DISCUSSION TITLE
我把它修改成中文版，并做了调整

### DISCUSSION THREAD

**Discussion by yozooCN** [99natmar99/Windows-11-Fixer#12] [Show and tell]
我把它修改成中文版，并且做了一些调整
- 原版不能自由调节窗口大小，且窗口大小固定为1280x720，我电脑显示区域有限，把窗口大小改小了
- 原版字体只支持英文显示，中文字体显示不全，字体修改为微软雅黑
- 配色改为较为领先的iPhone 13 Pro新配色苍岭绿

中文用户可以来下载和提意见
https://github.com/yozooCN/Windows-11-Fixer
![截屏2022-03-16 20 12 39](https://user-images.githubusercontent.com/76514453/158780811-aa7ecdec-d439-486c-ae46-119994000ab0.png)

**Comment by 99natmar99** [comment]
Thanks for taking the time to produce a Chinese version of the program.

感谢您花时间制作该程序的中文版。
